go/services/reports/pkg/config/modules.go

```diff
@@ -0,0 +1,189 @@
+package config
+
+import (
+	"fmt"
+	"github.com/Magpie-Monitor/magpie-monitor/pkg/elasticsearch"
+	sharedrepositories "github.com/Magpie-Monitor/magpie-monitor/pkg/repositories"
+	// "github.com/Magpie-Monitor/magpie-monitor/pkg/routing"
+	"github.com/Magpie-Monitor/magpie-monitor/pkg/tests"
+	"github.com/Magpie-Monitor/magpie-monitor/services/reports/internal/brokers"
+	"github.com/Magpie-Monitor/magpie-monitor/services/reports/internal/database"
+	"github.com/Magpie-Monitor/magpie-monitor/services/reports/internal/handlers"
+	"github.com/Magpie-Monitor/magpie-monitor/services/reports/internal/services"
+	incidentcorrelation "github.com/Magpie-Monitor/magpie-monitor/services/reports/pkg/incident_correlation"
+	"github.com/Magpie-Monitor/magpie-monitor/services/reports/pkg/insights"
+	"github.com/Magpie-Monitor/magpie-monitor/services/reports/pkg/openai"
+	"github.com/Magpie-Monitor/magpie-monitor/services/reports/pkg/repositories"
+	"go.uber.org/fx"
+	"go.uber.org/fx/fxevent"
+	"go.uber.org/zap"
+	"os"
+)
+
+var AppModule fx.Option
+
+func init() {
+	env := os.Getenv("APP_ENV")
+	fmt.Printf("Starting the app in %s environment", env)
+
+	if env == tests.TEST_ENVIRONMENT {
+		AppModule = fx.Options(
+			fx.WithLogger(func(log *zap.Logger) fxevent.Logger {
+				return &fxevent.ZapLogger{Logger: log}
+			}),
+			fx.Provide(
+
+				// routing.NewRootRouter,
```

Comment:
Are those comments needed?

Reply:
No, they are not, thanks for catching it